### ISSUE TITLE
BUG：评测loss计算中attention_mask有误

### ISSUE THREAD

**Issue by zhangbin1997** [SkyworkAI/Skywork#55]
你好，我看到你们的 eval_loss.py 中计算loss的时候，attention_mask 取的是 attention_mask[:, :-1]，同时是采用的 right padding。这样的做法在batch情况下，即有padding的时候，会导致非最大长度的那些句子会额外在最后计算一个生成padding_token的loss，并且该loss的数量级通常大于正文的数量级，会造成结果有误。在right padding的前提下，改成 attention_mask[:, 1:] 即可解决该问题。

**Comment by zhao1iang** [comment]
您好，感谢您的反馈，这个是上次修复issue不小心引入的bug，目前已解决，最近比较忙回复较晚请见谅。
